src/comgr.cpp

```diff
@@ -217,6 +219,28 @@ static void RemoveOptionsUnwanted(OptionList& list)
 
 namespace hip {
 
+static bool IsPchEnabled()
+{
+    if(miopen::IsEnabled(MIOPEN_DEBUG_COMGR_HIP_PCH_ENFORCE{}))
+        return true;
+    if(miopen::IsDisabled(MIOPEN_DEBUG_COMGR_HIP_PCH_ENFORCE{}))
+        return false;
+    return HIP_SUPPORTS_PCH;
+}
+
+static std::string GetPchEnableStatus()
```

Comment:
I feel like this could be useful elsewhere and not just logging.

Reply:
Only here so far.